src/views/ServerStartView.vue

```diff
@@ -2,21 +2,50 @@
   <div
     class="font-sans flex flex-col justify-center items-center h-screen m-0 text-neutral-300 bg-neutral-900 dark-theme pointer-events-auto"
   >
-    <h2 class="text-2xl font-bold">{{ ProgressMessages[status] }}</h2>
+    <div
+      v-if="status == ProgressStatus.ERROR"
+      class="flex flex-col items-center my-[60px]"
+    >
+      <h1 class="text-[60px] my-0">{{ $t('serverStartView.errorTitle') }}</h1>
+      <h2 class="my-[30px]">{{ $t('serverStartView.errorDescription') }}</h2>
```

Comment:
I think it fine to keep all buttons on the same row.

Reply:
Figma design specifies buttons in two rows

Reply:
Same as above on figma design scope.